index.js

```diff
@@ -59,7 +59,11 @@ exports.create = function (config) {
         renderer.putRaw = function (rawCssRule) {
             if (process.env.NODE_ENV === 'production') {
                 var sheet = renderer.sh.sheet;
-                sheet.insertRule(rawCssRule, sheet.cssRules.length);
+                try {
```

Comment:
I would like to approach avoid using `try/catch` in production for performance reasons.

It uses to have a `try/catch` block in development mode to catch these kind of errors and notify developer in development mode. See https://github.com/streamich/nano-css/blame/5f629b8d6ac47d029374ffd49ca77629f7efb23e/index.js#L63

I am thinking we revert back something like that. This way it will be fast in production and if there are any invalid styles, developer would see warnings in console.

Reply:
Maybe in dev mode there could be two style sheets, one in which styles are inserted using `.insertRule` &mdash; we would use that to catch any errors. And another one inserting using `.appendChild`, which is slower but renders better in Chrome dev tools.

Reply:
The problem is that these exceptions will only trigger in certain browsers. The last thing you want is things breaking only in production and only in browsers not commonly used for development.